spec/project/default_config_spec.rb

```diff
@@ -38,7 +41,17 @@ def cop_configuration(config_key)
   end
 
   it 'has configuration for all cops' do
-    expect(default_config.keys.sort).to eq(config_keys.sort)
+    expect(default_config.keys).to match_array(config_keys)
+  end
+
+  it 'sorts configuration keys alphabetically' do
+    namespaces.each do |_path, prefix|
+      expected = config_keys.select { |key| key.start_with?(prefix) }.sort
+      actual = default_config.keys.select { |key| key.start_with?(prefix) }
+      actual.each_with_index do |key, idx|
+        expect(key).to eq expected[idx]
```

Comment:
I tried to come up with a better way to set an expectation (`expect(actual).to match(expected)`) but I must admit that your matcher will give the most useful message in case of mismatch.

Reply:
I borrowed it from Rubocop :blush: